CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
-# Lowest version - android ndk 3.6.0
-cmake_minimum_required(VERSION 3.6.0)
+cmake_minimum_required(VERSION 3.5.1)
```

Comment:
Can you comment why we need this version 3.5.1 is :(

Reply:
I assume you mean in the code?  Added a comment to CMakeLists.txt.